### PR TITLE
Connection timeout fixes

### PR DESCRIPTION
**Pull request by bethrichardson**
Please review @fobofindia09 Please don't merge before talking with me.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 *.class
 .idea/
 *.iml
+/target/
```
Comment (recreationalcode) [inline .gitignore:4]: what does this do?

File: pom.xml [hunk 2]
```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
```
Comment (recreationalcode) [inline pom.xml:2]: What do changes to this file do?

File: .gitignore (hunk 1 above)
Comment (bethrichardson) [inline .gitignore:4]: ignores target when pushing to git

File: pom.xml (hunk 2 above)
Comment (bethrichardson) [inline pom.xml:2]: set target compiler mostly

**Comment by bethrichardson** [comment]
Merging per request by @fobofindia09

**Pull request merged**